Wrappers/Python/test/test_DataContainer.py

```diff
@@ -176,14 +175,8 @@ def test_unary_operations(self):
         self.assertEqual(ds.as_array()[0][0][0],
                          numpy.sqrt(2., dtype='float32'))
 
-    def test_binary_operations(self):
-        self.binary_add()
-        self.binary_subtract()
-        self.binary_multiply()
-        self.binary_divide()
-
-    def binary_add(self):
-        X, Y, Z = 512, 512, 512
+    def test_binary_add(self):
+        X, Y, Z = 128, 512, 512
```

Comment:
lowered to 128 from 512

Reply:
probably could go a lot smaller. We only need  to check on tiny arrays.